server/graphql/loaders/expenses.ts

```diff
@@ -1,12 +1,43 @@
 import DataLoader from 'dataloader';
 
 import ACTIVITY from '../../constants/activities';
-import models, { Op } from '../../models';
+import models, { Op, sequelize } from '../../models';
 import { ExpenseAttachedFile } from '../../models/ExpenseAttachedFile';
 import { ExpenseItem } from '../../models/ExpenseItem';
+import { LEGAL_DOCUMENT_TYPE } from '../../models/LegalDocument';
 
 import { sortResultsArray } from './helpers';
 
+const THRESHOLD = 600e2;
+const {
+  requestStatus: { RECEIVED },
+} = models.LegalDocument;
+
+
+const userTaxFormRequiredBeforePaymentQuery = `
+  SELECT 
+    e."UserId" "userId", 
+    MAX(er.id) as "expenseId",
+    MAX(ld."requestStatus") as "legalDocRequestStatus",
+    MAX(d."documentType") as "requiredDocument",
+    SUM (e."amount") AS total
+  FROM "Expenses" e
+  INNER JOIN "Expenses" er ON er.id IN (:expenseIds)
+  INNER JOIN "Collectives" c ON c.id = e."CollectiveId"
+  LEFT JOIN "RequiredLegalDocuments" d ON d."HostCollectiveId" = c."HostCollectiveId"
```

Comment:
Small optimization: no need to check the expenses for hosts without required legal documents

```suggestion
  INNER JOIN "RequiredLegalDocuments" d ON d."HostCollectiveId" = c."HostCollectiveId"
```